.github/actions/deploy-environment/action.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy Environment
+
+inputs:
+  azure_credentials:
+    description: The GitHub access token
+    required: true
+  environment_name:
+    description: Name of the environment in Make file
+    required: true
+  root_folder:
```

Comment:
I understand this action is only for cluster, so we could hard code it in the action and remove the input

Reply:
It was intended to be generalisable on the assumption we'd deploy everything using a workflow in the future